dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/liberty/CDIRuntimeImpl.java

```diff
@@ -145,6 +147,12 @@ public class CDIRuntimeImpl extends AbstractCDIRuntime implements ApplicationSta
     private ProxyServicesImpl proxyServices;
 
     public void activate(ComponentContext cc) {
+        //This emmits logging in a static block.
+        //OpenTelemetry can have a circular dependency if that loging goes into OTel
+        //And the application calls CDI.current() during its OTel configuration extensions.
+        //So force it early
+        WeldStartup ws = new WeldStartup();
```

Comment:
If we don't already have one, we should add a test for logging from a portable extension with telemetry enabled.